ibc_proto/src/prost/ibc.channel.rs

```diff
@@ -25,8 +25,8 @@ pub struct MsgChannelOpenTry {
     pub counterparty_version: std::string::String,
     #[prost(bytes, tag="5")]
     pub proof_init: std::vec::Vec<u8>,
-    #[prost(uint64, tag="6")]
-    pub proof_height: u64,
+    #[prost(message, optional, tag="6")]
+    pub proof_height: ::std::option::Option<super::client::Height>,
```

Comment:
The `Height` as defined here is a tuple of two `u64`, so it's important to note that this will likely impact our code (e.g., the TryFromRaw) and other parts.

Reply:
Relevant: https://github.com/informalsystems/ibc-rs/issues/191